src/Header/ContentType.php

```diff
@@ -10,8 +10,9 @@
 namespace Zend\Mail\Header;
 
 use Zend\Mail\Headers;
+use Zend\Mime\Mime;
 
-class ContentType implements HeaderInterface
+class ContentType implements UnstructuredInterface
```

Comment:
I missed that the interface already exists. Ugh. Okay, thanks; will change to the 2.5 release.